src/domain/settings/regions/edit-shipping.tsx

```diff
@@ -0,0 +1,248 @@
+              <label className="inline-flex items-center inter-base-semibold">
+                <input
+                  type="checkbox"
+                  id="true"
+                  name="requires_shipping"
+                  value="true"
+                  checked={!adminOnly}
+                  onChange={() => setAdminOnly(!adminOnly)}
+                  className="mr-small w-5 h-5 accent-violet-60 rounded-base"
+                />
+                Show on website
+              </label>
+            </div>
+            <p className="inter-base-semibold mb-base">Requirements</p>
+            <div className="grid grid-cols-1 medium:grid-cols-2 gap-base">
+              <div className="flex items-center gap-2xsmall">
+                <Input
+                  label="Currency"
+                  value={region.currency_code.toUpperCase()}
+                  readOnly
+                  className="w-[120px] pointer-events-none"
+                  tabIndex={-1}
+                />
+                <Input
+                  label="Min. subtotal"
+                  type="number"
+                  name={`requirements.min_subtotal.amount`}
+                  min={0}
+                  ref={register}
+                  placeholder="0"
+                />
+              </div>
+              <div className="flex items-center gap-2xsmall">
+                <Input
+                  label="Currency"
+                  value={region.currency_code.toUpperCase()}
+                  readOnly
+                  className="w-[120px] pointer-events-none"
+                />
+                <Input
+                  label="Max. subtotal"
+                  type="number"
+                  min={0}
+                  name={`requirements.max_subtotal.amount`}
+                  ref={register}
+                  placeholder="100"
+                />
+              </div>
+            </div>
+            <div className="mt-xlarge">
+              <p className="inter-base-semibold">Danger Zone</p>
+              <p className="inter-base-regular text-grey-50 mb-base">
+                This will permanently delete this option from your Medusa Store
+              </p>
+              <button
+                onClick={handleDelete}
+                className="text-rose-50 inter-base-semibold"
+              >
+                Delete Option
+              </button>
+            </div>
+          </Modal.Content>
+          <Modal.Footer>
+            <div className="flex items-center justify-end w-full">
+              <Button
+                type="button"
+                onClick={onClick}
+                variant="ghost"
+                size="small"
+                className="w-[127px] justify-center"
+              >
+                Cancel Changes
+              </Button>
+              <Button
+                type="submit"
+                variant="primary"
+                size="small"
+                className="w-[127px] justify-center"
```

Comment:
This width should be added to tailwind-config or we should use `w-32` and accept that it's 128px instead

Reply:
Generally don't think we should use the built in spacings from Tailwind such as w-32, as we have implemented our own spacings according to Ludvig's design. I've added a w-eventButton in this case 😄

Reply:
perfect 😄